src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,19 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/mate";
+    private static final String DB_USER = "root";
+    private static final String DB_PASSWORD = "Qwerty123";
+
+    public static Connection getConnection() throws SQLException {
+        try {
+            return DriverManager.getConnection(DB_URL, DB_USER, DB_PASSWORD);
+        } catch (SQLException e) {
+            throw new RuntimeException("Cannot connect to database", e);
+        }
```

Comment:
Catching `SQLException` and re-throwing it as a `RuntimeException` is not the best practice because it removes the ability of the caller to handle database exceptions in a granular way. It's better to let the `SQLException` be thrown to the caller, or create a custom checked exception for database-related errors.